.github/workflows/release.yml

```diff
@@ -200,11 +200,22 @@ jobs:
       base64-subjects: "${{ needs.sbom-assests.outputs.hashes }}"
       provenance-name: "karmada-sbom.intoto.jsonl"
       upload-assets: true
+
   update-krew-index:
-    needs: release-assests
+    needs: 
+    - release-assests
     name: Update krew-index
     runs-on: ubuntu-22.04
     steps:
+    - name: get latest tag
+      id: get-latest-tag
+      run: |
+        export LATEST_TAG=`gh api repos/karmada-io/karmada/releases/latest | jq -r '.tag_name'`
+        echo "Got the latest tag:$LATEST_TAG"
+        echo "event.tag:"${{ github.event.release.tag_name }}
+        echo "latestTag=$LATEST_TAG" >> "$GITHUB_OUTPUT"
     - uses: actions/checkout@v4
+      if: steps.get-latest-tag.outputs.latestTag == github.event.release.tag_name
     - name: Update new version in krew-index
+      if: steps.get-latest-tag.outputs.latestTag == github.event.release.tag_name
```

Comment:
The two steps were skipped, I don't know if this workflow could get the right latest tag, maybe there is a syntax error?
https://github.com/karmada-io/karmada/actions/runs/10645084310/job/29510560441#step:3:3

I can get the latest tag on my side.
```
# gh api repos/karmada-io/karmada/releases/latest | jq -r '.tag_name'
v1.11.0

```